components/style/build_gecko.rs

```diff
@@ -250,7 +250,15 @@ mod bindings {
                 return;
             }
         }
-        let mut result = builder.generate().expect("Unable to generate bindings").to_string();
+        let command_line_opts = builder.command_line_flags();
+        let result = builder.generate();
+        let mut result = match result {
+            Ok(bindings) => bindings.to_string(),
+            Err(_) => {
+                println!("Failed to generate bindings, flags: {:?}", command_line_opts);
+                panic!("Unable to generate bindings");
```

Comment:
You can just `panic!("Failed to generate bindings, flags: {:?}", command_line_opts)`.